widgets/waterTracker/main.js

```diff
@@ -25,6 +25,8 @@ const resetBtn = document.getElementById('reset');
 const goalInput = document.getElementById('goalWater');
 const submit = document.getElementById('submit');
 const waterFilledElement = document.getElementById('water');
+const waterAmountModalBtn = document.getElementById('addAmountWater');
+const main = document.getElementsByClassName('main')[0];
 
```

Comment:
@sauravk7077 
Just to teach me... please...
I haven't seen an index following a getElementsBy statement.  Does it return an array of each class="main"?